### PR TITLE
Update Texas.md

### PR DESCRIPTION
**Pull request by jmc91**

### CODE REVIEW

File: reports/Texas.md [hunk 1]
```diff
@@ -302,3 +302,32 @@ id: tx-sanantonio-2
 **Links**
 
 * https://twitter.com/SanAntonioProbz/status/1268027491987468288
+
+### Police gang up on protestors to beat them with batons. Then pepper spray man for recording | (believed to be) June 2nd
```
Comment (emarcey) [inline reports/Texas.md:306]: Can you put this under Houston?

File: reports/Texas.md [hunk 2]
```diff
@@ -302,3 +302,32 @@ id: tx-sanantonio-2
...
+
+tags: baton, pepper-spray, punch
+
+id: tx-houston-3
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1268392253665546245
+
+### Police fire teargas canisters and projectiles into peaceful crowd of unarmed protestors | (believed to be) May 31st
```
Comment (emarcey) [inline reports/Texas.md:318]: Can you put this under El Paso? If it doesn't exist can you make a new El Paso alphabetically ordered

Comment (jmc91) [reply reports/Texas.md:318]: ok, I see now. yep will re-order these

Comment (jmc91) [reply reports/Texas.md]: @emarcey done

File: reports/Texas.md [hunk 3]
```diff
@@ -274,6 +288,18 @@ id: tx-houston-3
 * https://twitter.com/DomSkyeRN/status/1270185401060294656
 
 
+### Police gang up on protestors to beat them with batons. Then pepper spray man for recording | (believed to be) June 2nd
+
+Police isolate protester and then 4 to 5 officers beat him with a baton.  When one of them realizes he is being recorded, he pepper sprays the person recording to make him stop.
```
Comment (emarcey) [inline reports/Texas.md:293]: A little disconnect between the title & description. I think change the title to say they're beating one person

**Pull request merged**